bindings/nodejs/lib/types/client/build-block-options.ts

```diff
@@ -31,5 +31,5 @@ export interface IBuildBlockOptions {
 /** Address with base coin amount */
 export interface IClientBlockBuilderOutputAddress {
     address: string;
-    amount: string;
+    amount: bigint | string;
```

Comment:
should this be hexencodedamount, or jsut string is fine?

Reply:
This is a decimal string, not hex, should we actually make a type for that too? Something like `NumericString`?

Reply:
i think we should yea, to avoid confusion as currently its still not very clear what is what NumericString sounds great

Reply:
✅ Will do in separate PR